src/esw/brushless.py

```diff
@@ -244,20 +238,25 @@ async def update(self) -> None:
             await self._connect()
 
 
-class MotorsManager:
+class MotorsManager(ABC):
     BASESTATION_TO_ROVER_NODE_WATCHDOG_TIMEOUT_S = 1
 
-    def __init__(self, manager_name: str, motor_controller_info_by_name, transport, publish_topic: str):
-        self._manager_name = manager_name
-        self._motor_bridge_by_name = {}
+    _motor_names: List[str]
+    _motor_bridge_by_name: Dict[str, MoteusBridge]
+    _command_data: List[CommandData]
+
+    def __init__(self, motor_controller_info_by_name: dict, transport):
         self._motor_names = []
+        self._motor_bridge_by_name = {}
+        self._command_data = []
         for name, info in motor_controller_info_by_name.items():
             self._motor_names.append(name)
             self._motor_bridge_by_name[name] = MoteusBridge(info["id"], transport)
+            self._command_data.append(CommandData())
 
         self._last_updated_time = t.time()
 
-        self._motors_status_publisher = rospy.Publisher(publish_topic, MotorsStatus, queue_size=1)
+        self._motors_status_publisher = rospy.Publisher(self.publish_topic(), MotorsStatus, queue_size=1)
```

Comment:
so the bash style.sh complains on this line, idk why.

Reply:
I think since it's a property, it wants to not treat it like a function. idk if there's a better alternative or not

Reply:
Ok i made it pass... basically i just had to add @property to the classes that it was referring to as well

Reply:
Ah okay that makes sense